coralproject/talk
# [CORL-1417] Fix issue with external images not displaying consistently in admin

## What does this PR do?

There was a strange bug causing images to appear and disappear in the admin. It turned out to be related to a memoized ID value that ended up being the same value between multiple instances of a component, causing it to only be mounted once, so it would disappear from one place in the UI and re-appear in another. This PR appends a UUID to the ID to ensure each component has a unique ID even if two of them have the same image ID. 



## What changes to the GraphQL/Database Schema does this PR introduce?
none


## How do I test this PR?

create a comment with an external image. View the comment in teh admin. open the user drawer or conversation modal and ensure the images do not disappear.
